src/main/resources/openapiv2.json

```diff
@@ -859,27 +1035,111 @@
+                },
+                "key": {
+                    "type": "object"
+                }
+            }
+        },
+        "ProducerRecordList": {
             "title": "Records",
             "type": "object",
             "properties": {
                 "records": {
-                    "$ref": "#/definitions/Records"
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ProducerRecord"
+                    }
+                }
+            },
+            "example": {
+                "records": [
+                    {
+                        "key": "1abcd",
+                        "value": "msg1"
+                    },
+                    {
+                        "value": "msg2",
+                        "partition": 1
+                    },
+                    {
+                        "value": "msg3"
+                    }
+                ]
+            }
+        },
+        "ErrorBody": {
+            "title": "Root Type for ErrorBody",
+            "description": "The root of the ErrorBody type's schema.",
+            "type": "object",
+            "properties": {
+                "error_code": {
+                    "format": "int32",
+                    "type": "integer"
+                },
+                "message": {
+                    "type": "string"
                 }
             },
-            "example": "{\n    \"records\": [\n        {\n            \"key\": \"1abcd\",\n            \"value\": \"msg1\"\n        },\n        {\n            \"value\": \"msg2\"\n        }\n    ]\n}"
+            "example": "{\n    \"error_code\": 404,\n    \"message\": \"resource not found\"\n}"
         }
     },
     "tags": [
         {
             "name": "Consumers",
-            "description": "The `consumers` resource allows you to create consumers in your Kafka cluster and perform common actions, such as subscribing to topics, retrieving processed records, and committing offsets."
+            "description": "The `consumers` endpoints allow you to create consumers in your Kafka cluster and perform common actions, such as subscribing to topics, retrieving processed records, and committing offsets."
         },
         {
             "name": "Topics",
-            "description": "The `topics` resource allows you to send messages to a specified topic or topic partition. You can include message keys in requests."
+            "description": "The `topics` endpoints allow you to send messages to a specified topic or topic partition. You can include message keys in requests."
+        },
+        {
+            "name": "Seek",
+            "description": "`Seek` related operations enable a consumer to begin receiving messages from a given offset position."
+        },
+        {
+            "name": "Producer",
+            "description": "The `producer` endpoints allow you to send records to a specified topic or topic partition."
         }
```

Comment:
Regarding all the tags descriptions I would use the same form of the "Seek" one.
The tags are a way of grouping things. So the "Seek" description says clearly "... related operations ..." so it describe that it's grouping operation related to seek.
I would use the same for the other tags. Saying "The consumers endpoints ..." for example seems to me that we are describing an endpoint and not a tag. @PaulRMellor help on this please.

Reply:
Consumer operations to create consumers in your Kafka cluster and perform common actions, such as subscribing to topics, retrieving processed records, and committing offsets.

Producer operations to send records to a specified topic or topic partition.

Seek operations that enable a consumer to begin receiving messages from a given offset position.

Topic operations to send messages to a specified topic or topic partition. You can include message keys in requests.